cli/flox/src/utils/init/sentry.rs

```diff
@@ -26,20 +32,11 @@ pub fn init_sentry() -> Option<ClientInitGuard> {
         .clone()
         .unwrap_or_else(|| "development".to_string());
 
-    // TODO: configure user
-    // https://docs.sentry.io/platforms/rust/enriching-events/identify-user/
-    // sentry::configure_scope(|scope| {
-    //     scope.set_user(Some(sentry::User {
-    //     ..
-    //    }));
-    // });
-
     let sentry = sentry::init(sentry::ClientOptions {
         dsn: Some(sentry_dsn),
 
         // https://docs.sentry.io/platforms/rust/configuration/releases/
-        // TODO: should we maybe just use commit hash
-        release: sentry::release_name!(),
+        release: Some(Cow::Borrowed(&FLOX_VERSION)),
```

Comment:
Yep:

> So that it's consistent with the versions that we use for:
> 
> - `flox --version`
> - in-house metrics
> - HTTP user agents
> 
> This also ensure that we set a real value when used outside of an
> installer, e.g. flake and devShell.

I suspect it did work prior to a9d6092d4645150ed61dfd1ebc772fea356b798f because [it uses CARGO_PKG_VERSION](https://docs.rs/sentry-core/0.34.0/src/sentry_core/macros.rs.html#18) and [that would have been set automatically](https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates).

I've just noticed that one difference from `release_name!` is that we no longer prefix it with `flox@`. We could add that, or `flox-cli@`, to be consistent with the other places.

Reply:
might be worth checking if sentry expects this in that format
if there is a soft dependecy on that i think its worth kepping the format

Reply:
[Before](https://flox-dev.sentry.io/performance/trace/d3faa729b3df5d20b8204fc05221e5af/?node=trace-root&pageEnd&pageStart&project=4506548241825792&query=project%3Acli-flox+user.id%3Ac3318f2c-9e7a-4377-ac81-161ae892fe79&source=traces&statsPeriod=1h&timestamp=1724424455.549&utc=true):

![image](https://github.com/user-attachments/assets/7ba3ead5-43ed-4473-99e5-99eed7f984d2)

[After](https://flox-dev.sentry.io/performance/trace/4481647df77a51ca2a50b12419aa1c87/?node=trace-root&pageEnd&pageStart&project=4506548241825792&query=project%3Acli-flox+user.id%3Ac3318f2c-9e7a-4377-ac81-161ae892fe79&source=traces&statsPeriod=1h&timestamp=1724424235.941&utc=true):

![image](https://github.com/user-attachments/assets/af3ed9d2-b040-450c-b592-70de95aa86ae)

It now appears correctly in the [releases list](https://flox-dev.sentry.io/releases/?project=4506548241825792&query=1.3.0&statsPeriod=1h&utc=true):

![image](https://github.com/user-attachments/assets/e09b0730-3de1-4e81-9897-d1298017cc29)

Confusingly it still renders the tag as the version only: 
![image](https://github.com/user-attachments/assets/3906a535-d888-465d-9d0f-1bb5c5bb181e)

Reply:
id say keep the `@` syntax then, since sentry does seem to parse that

Reply:
It's done in 3d76eae 👍🏻